### PR TITLE
Small overview

### PR DESCRIPTION
**Pull request by kousu**
I made some updates here. I left some links blank because I was just doing this quickly, please fill them in if you have a moment.

### CODE REVIEW

**Comment by mdosseva** [comment]
@kousu you are a human presumably, and not a spambot of love, which is a terrible shame because I would prefer the latter.

**Comment by kousu** [comment]
Too bad so sad

Le 21 avril 2017 15:27:12 HAE, mdosseva <notifications@github.com> a écrit :
>@kousu you are a human presumably, and not a spambot of love, which is
>a terrible shame because I would prefer the latter.
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/mdosseva/roofs4refugees/pull/1#issuecomment-296285722

-- 
kousu

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+the Wordpress plugin [FS Contact Form](TODO) builds forms which are used to take submissions of various kinds
+of offers of help.
+[osTicket](TODO) sits on admin.roofs4refugees.ca and receives the form submissions _via email_, reparses them,
+and populates its database.
+
+## Plan
+
+1. ???
+2. ???
+3. Non-profit!
```
Comment (mdosseva) [inline README.md]: lawls

Comment (mdosseva) [inline README.md]: lawls

**Pull request merged**